doc/api/tracing.md

```diff
@@ -27,7 +29,32 @@ The available categories are:
 By default the `node`, `node.async_hooks`, and `v8` categories are enabled.
 
 ```txt
-node --trace-events-enabled --trace-event-categories v8,node,node.async_hooks server.js
+node --trace-event-categories v8,node,node.async_hooks server.js
+```
+
+Prior versions of Node.js required the use of the `--trace-events-enabled`
+flag to enable trace events. This requirement has been removed. However, the
+`--trace-events-enabled` flag *may* still be used and will enable the
+`node`, `node.async_hooks`, and `v8` trace event categories by default.
```

Comment:
Not a feedback on this PR, but I don't think `node.async_hooks` are good candidates for the default set of events to enable. They are a bit too low level. Higher level tracing of at the APIs level would be a lot more useful to most users. Since `trace_events` is experimental, I we should be able to change this in the future.

Reply:
I think it is for backward compatibility, ultimately I think `--trace-events-enabled` should be removed. A default set of `trace_events` doesn't really make sense to me.

Reply:
Yes, it's for backwards compatibility only. Once this lands I plan to open a PR deprecating `--trace-events-enabled` and removing it from the `node --help` output. It will still be there for legacy support and can eventually be removed.

Reply:
sgtm